src/login/login.component.html

```diff
@@ -29,18 +29,18 @@
 
           <md-input-container class="md-icon-float md-icon-left"> <!--TODO: md-icon-left is a temp fix, remove once AM fixes this-->
             <!-- Use floating label instead of placeholder -->
-            <label>Username</label>
+            <label>Email</label>
             <md-icon>person</md-icon>
             <input ng-disabled="loginForm.$submitted"
-                   ng-model="ctrl.credentials.username"
+                   ng-model="ctrl.credentials.email"
                    ng-change="loginForm.password.$setValidity('auth', true)"
-                   id="username"
-                   name="username"
+                   id="email"
+                   name="email"
```

Comment:
Is there any validation if it is an email? There's an `email` input type.

Reply:
Missed that, thanks